projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -336,6 +336,10 @@ public function add_jetpack_menu() {
 		// This is supposed to be the same as class-admin-menu but with a different position specified for the Jetpack menu.
 		if ( 'wp-admin' === get_option( 'wpcom_admin_interface' ) ) {
 			parent::create_jetpack_menu( 2, false );
+
+			// Remove the Newsletter submenu item from Settings and add it under Jetpack menu.
+			remove_submenu_page( 'options-general.php', 'https://wordpress.com/settings/newsletter/' . $this->domain );
```

Comment:
Do you think we should wrap [this line](https://github.com/automattic/jetpack/blob/trunk/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L384) with the wp-admin check instead or do you prefer keeping this code together?

Reply:
Does this change apply to Simple sites as well? If so, I think we should move this check to the base class.

Reply:
Yes, it is related to where to put the logic:

- Applicable to both Atomic and Simple? Wrap the `add_submenu_page()` with the check in the admin-menu class
- Applicable to only Atomic? Use `remove_submenu_page()` in the atomic-admin-menu class

Reply:
True, I did it thinking only on Atomic, should move it for Simple+Atomic?

Reply:
I updated with your suggestions

Reply:
Same as https://github.com/Automattic/jetpack/pull/35429#discussion_r1480791682 classic view only for now.